docs/ADOPTING_DARIA.md

```diff
@@ -0,0 +1,43 @@
+# Adopting DARIA
+
+If you are a fund considering adopting DARIA as your case management system of record, this may answer common questions about how things work. We also have a sandbox/training app where no actual patient data lives for experimenting. [FILL IN ANY NECESSARY DETAIL]
+
+## The case management workflow
+
+DARIA's case management workflow was designed to mirror existing practices used in case management. Volunteer case managers answer patient calls coming in through a hotline phone number and voicemail. DCAF case management software helps the case managers keep track of patient status and needs.
+
+The workflow is similar to many Customer Relationship Manager (CRM) systems. After patients leave a voicemail, the case managers return their calls and assist patients in navigating the process of scheduling an appointment and raising funds. If the patient is short on money, after a series of phone calls we issue a pledge to support their visit to a clinic. If a patient takes advantage of the pledge and complete their abortion, our accounting team logs that as a fulfillment.
+
+Read more about our workflow at: 
+https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/DCAF_101.md
+
+## Administration toolkit
+
+DARIA provides a number of workflows to administer and report on your funds activities. Analysts are able to export an anonymized spreadsheet for analysis; administrators are able to keep track of which clinics your fund  are and aren't working with; and your accounting team can use a specialized workflow to mark which pledges have been cashed.
+
+## Your DARIA instance and its data is yours
+
+Every fund's DARIA instance is separate from others. Your DARIA instance and its data is completely yours to use. DCAF and other DARIA funds do not have access to your data.
```

Comment:
For an overview of the exact data that the DARIA app collects, please see:  https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/DATA_DICTIONARY.md.

(also, I love that y'all made the data dictionary, really helpful, def want to copy this in my other projects)

Reply:
oh... actually I think the suggested data dictionary verbiage belongs in the administration toolkit section.

Reply:
Particulars of data collection feels a little weedy to me as well -- is this something that other funds have both asked about in initial conversations with you? Other funds who have adopted this so far have not really been that interested in an analytics program and were much more interested in getting out of spreadsheet world, but I'm interested in whether that continues to hold true.

Reply:
This has come up in conversations about DARIA less in the context of an analytics program but more in the context of how similar is DARIA's intake to their intake and the data that they're already collecting and using for their process.